src/components/landingpage/styles.js

```diff
@@ -3,12 +3,12 @@ import imagesStyle from "../shared/styles/imagesStyles";
 
 const landingPageStyle = {
   "app-container": {
-    paddingTop: "0px"
+    paddingTop: "0px",
   },
   container: {
     zIndex: "12",
     color: "#FFFFFF",
-    ...container
+    ...container,
```

Comment:
In some areas, you're leaving a trailing comma here and there, wherein other areas you're not. I'd recommend some static analysis testing be added (ESLint + Prettier formatting tool) to this repo to become more uniform.